core/src/com/raccoon/mygame/models/Guard.java

```diff
@@ -1,5 +1,68 @@
 package com.raccoon.mygame.models;
 
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.*;
+import com.badlogic.gdx.graphics.*;
+import com.raccoon.mygame.view.GameCanvas;
+
 public class Guard {
+    private Vector2 position;
+
+    private float width;
+    private float height;
+
+    private Vector2 velocity;
+
+    private Texture patrolTexture;
+
+    Guard(float x, float y, float width, float height, Texture texture){
+        position = new Vector2(0.0f, 0.0f);
+        velocity = new Vector2(0.0f, 0.0f);
+        this.width = width;
+        this.height = height;
+        patrolTexture = texture;
+        position = new Vector2(x, y);
```

Comment:
you say position = new Vector2() twice. You should only need the last one where you set it equal to x and y.

Reply:
oops thanks for catching that, will make a commit to get rid of the last position definition

Reply:
> you say position = new Vector2() twice. You should only need the last one where you set it equal to x and y.

just fixed, commited and pushed!